src/app/components/Curation/getComponentName/index.ts

```diff
@@ -11,6 +10,7 @@ export const COMPONENT_NAMES = {
   HIERARCHICAL_CURATION_GRID: 'hierarchical-curation-grid',
   NOT_SUPPORTED: 'not-supported',
```

Comment:
Curious, but when do we actually return 'not-supported' IRL?

Reply:
We used to return not supported for curations with visual style = BANNER and visual prominence = MAXIMUM (see  https://github.com/bbc/simorgh/pull/11323 - which meant we did not render the component. But now we do!

Reply:
Ah yeah I was wondering that - for the future Billboard. Cool